runtime/bcverify/staticverify.c

```diff
@@ -1767,6 +1767,24 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 			goto _formatError;
 		}
 
+		/* The requirement for taking no arguments was introduced in Java SE 9.
+		 * In a class file whose version number is 51.0 or above, the method
+		 * has its ACC_STATIC flag set and takes no arguments.
+		 */
+                 /* Leave this here to find usages of the following check:
+                  * if (J2SE_VERSION(vm) >= J2SE_19) {
+                  */
+		if (((vmVersionShifted & BCT_MajorClassFileVersionMask) >= BCT_Java9MajorVersionShifted) || ((vmVersionShifted & BCT_MajorClassFileVersionMask) == 0)) {
```

Comment:
`vmVersionShifted & BCT_MajorClassFileVersionMask` can this be pulled into a local?  The check will more readable that way.

Reply:
Or, rather than passing the shifted version around, can the version be determined at the calcite and just pass the vmSupportedVersion around?

Reply:
The mask is redundant from the declaration of vmVersion in cfreader.c. I'll remove it.